Tools/AP_Bootloader/can.cpp

```diff
@@ -85,11 +85,12 @@ static struct {
 
 enum {
     FAIL_REASON_NO_APP_SIG = 10,
-    FAIL_REASON_BAD_LENGTH = 11,
+    FAIL_REASON_BAD_LENGTH_APP = 11,
     FAIL_REASON_BAD_BOARD_ID = 12,
     FAIL_REASON_BAD_CRC = 13,
     FAIL_REASON_IN_UPDATE = 14,
     FAIL_REASON_WATCHDOG = 15,
+    FAIL_REASON_BAD_LENTGH_DESCRIPTOR = 16,
```

Comment:
spelling error

Reply:
omg, I just triple checked DESCRIPTOR and was about to comment that you were wrong but just now realized it's LENTGH that was misspelled! LMAO Ok, fixing now

Reply:
fixed, changed to FAIL_REASON_BAD_LENGTH_DESCRIPTOR